openstack/octavia/templates/etc/_secrets.conf.tpl

```diff
@@ -22,3 +22,7 @@ record_payloads = {{ if .Values.audit.record_payloads -}}True{{- else -}}False{{
 metrics_enabled = {{ if .Values.audit.metrics_enabled -}}True{{- else -}}False{{- end }}
 {{- include "ini_sections.audit_middleware_notifications" . }}
 {{- end }}
+
+{{- if .Values.osprofiler.enabled }}
```

Comment:
Please keep the `# Tracing` comment (unless of course it's somehow not correct). Thanks!